.github/actions/configure-cmake-project/action.yml

```diff
@@ -461,7 +461,8 @@ runs:
           }
         }
 
-        if ($EnableCaching) {
+        # TODO(steelskin): Figure out why sccache is broken with "-gsplit-dwarf"
```

Comment:
`sccache` does not support `-gsplit-dwarf` as there are multiple outputs and it does not know how to map that I believe.

Reply:
I renamed the comment, but left the TODO. I don't think this is a huge deal in terms of lost build time, but it'd still be nice to have.